Project-Code/src/index.js

```diff
@@ -88,14 +97,11 @@ app.get('/test', (req, res)=>{
                 location: results.data.pollingLocations
             });
          })
-         .catch(error => {
-        //hanlde errors
-        res.render({
-        results: [],
-        error: error,
-        })
-    })
-});
+        .catch((error) => {
+            //hanlde errors
+            console.log(error);
```

Comment:
handle the error with a response? send back an error, probably just a 200 response still i'm guessing